### PR TITLE
Removes expired elements before accessing elements.

### PR DESCRIPTION
**Pull request by brian-js**
[Review on Reviewable](https://reviewable.io/reviews/maidsafe/lru_time_cache/38)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @dirvine

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: src/lib.rs [hunk 1]
```diff
@@ -163,26 +166,23 @@ impl<K, V> LruCache<K, V> where K: PartialOrd + Ord + Clone, V: Clone {
...
     }
 
-    /// Check for existence of a key
-    // FIXME: Deprecated in favor of the `contains_key` function defined below.
-    pub fn check(&self, key: &K) -> bool {
-        self.map.contains_key(key)
-    }
-
     /// Returns true if a value existed for the specified key.
-    pub fn contains_key(&self, key: &K) -> bool {
```
Comment (dirvine) [inline src/lib.rs]: much better (and faster)

File: src/lib.rs [hunk 2]
```diff
@@ -163,26 +166,23 @@ impl<K, V> LruCache<K, V> where K: PartialOrd + Ord + Clone, V: Clone {
         })
     }
 
-    /// Check for existence of a key
-    // FIXME: Deprecated in favor of the `contains_key` function defined below.
```
Comment (dirvine) [inline src/lib.rs]: As this is an API change we will need to publish a release version change when this is merged.

Comment (brian-js) [inline src/lib.rs]: Yeah, I took the opportunity to clean it up here. Do you want me to raise a version change PR?

**Pull request merged**